crates/worker/src/partition/state_machine/mod.rs

```diff
@@ -1147,10 +1147,26 @@ impl<S> StateMachineApplyContext<'_, S> {
                 InvocationStatus::Invoked(_) | InvocationStatus::Suspended { .. }
             ) =>
             {
-                // We ignore the cancel when we haven't a pinned deployment yet,
-                // because we don't know what we should do (cancellation works differently between journal table v1 and v2).
-                // In any case, this would have no visible effect to the user
-                info!("Ignoring cancellation because the invocation made no progress so far!");
+                // We need to apply a corner case fix here.
+                // We don't know yet what's the protocol version being used, but we know the status is either invoker or suspended.
+                // To sort this out, we write a field in invocation status to
```

Comment:
to what? Seems that some words are missing.

Reply:
a common case of starting a sentence without finishing it :D